lf-to-abstract-sql.ometajs

```diff
@@ -872,8 +872,8 @@ export ometa LF2AbstractSQL <: SBVRCompilerLibs {
 		]
 		{{}}:hasDependants
 		{
-			_.each(this.tables, function(table) {
-				_.each(table.fields, function(field) {
+			_.forEach(this.tables, function(table) {
```

Comment:
Did you mean

```suggestion
			this.tables.forEach(function(table) {
```

here and below?

Reply:
Nope, this is an object - I included it as standardizing on forEach to match the native version